python/GafferSceneUI/_InspectorColumn.py

```diff
@@ -420,11 +420,85 @@ def __keyPress( column, pathListing, event ) :
 
 	return False
 
+def __dragEnter( path, pathListing, event ) :
+
+	column = pathListing.columnAt( event.line.p0 )
+	if path is None or not isinstance( column, GafferSceneUI.Private.InspectorColumn ) :
```

Comment:
We should be guaranteed to have an InspectorColumn shouldn't we, since we only connect to signals on InspectorColumns?

Reply:
Yep, this check is redundant. Removed in 71166ddccded22621b83f45bfbd3d27e2a72fe56.